tests/spec/smoketest-eq-services-spec.js

```diff
@@ -0,0 +1,68 @@
+const AuthorHome = require('../pages/AuthorHome.page');
+const CreateQuestionnairePage = require('../pages/CreateQuestionnaire.page');
+const DesignQuestionnairePage = require('../pages/DesignQuestionnaire.page');
+
+const BASE_URL = process.env.BASE_URL || 'http://localhost:3000';
+
+describe('eQ Services Smoke Test', () => {
+  beforeAll(() => {
+    browser.url(BASE_URL);
+    browser.click(AuthorHome.createQuestionnaireButton());
+    browser.setValue(CreateQuestionnairePage.questionnaireTitle(), 'title');
+    browser.setValue(CreateQuestionnairePage.questionnaireDescription(), 'description');
+    browser.selectByVisibleText(CreateQuestionnairePage.questionnaireTheme(), 'default');
+    browser.selectByVisibleText(CreateQuestionnairePage.questionnaireLegalBasis(), 'StatisticsOfTradeAct');
+    browser.click(CreateQuestionnairePage.navigationToggle());
+    browser.click(CreateQuestionnairePage.createButton());
+  });
+
+  it('should update navigation title when section title changed', () => {
+    browser.waitForExist("h3[class*='SectionTitle']");
+    expect(browser.getText(DesignQuestionnairePage.getFirstSectionTitle())).toEqual('Section Title');
+
+    // Adds section title
+    browser.setValue(DesignQuestionnairePage.setSectionTitle(), 'eQ Services Smoke Test');
+    browser.keys('Tab');
+    browser.waitForExist("h3[class*='SectionTitle']");
+    expect(browser.getText(DesignQuestionnairePage.getFirstSectionTitle())).toEqual('eQ Services Smoke Test');
+
+    // Adds first question
+    browser.setValue(DesignQuestionnairePage.setQuestionTitle(), 'Test Question 1');
+    browser.keys('Tab');
+    browser.click(DesignQuestionnairePage.clickAddAnswer());
+    browser.waitForExist("button[title='Text']");
+    browser.click(DesignQuestionnairePage.selectTextFieldAnswer());
+    browser.waitForExist("#question-page-editor [id='label']");
+    browser.setValue(DesignQuestionnairePage.setAnswerTitle(), 'Test Answer 1');
+
+    // Adds second Question
+    browser.click(DesignQuestionnairePage.clickAddPage());
+    browser.waitForExist("h3[class*='SectionTitle']");
+    browser.setValue(DesignQuestionnairePage.setQuestionTitle(), 'Test Question 2');
+    browser.keys('Tab');
+    browser.click(DesignQuestionnairePage.clickAddAnswer());
+    browser.waitForExist("button[title='Text']");
+    browser.click(DesignQuestionnairePage.selectTextFieldAnswer());
+    browser.waitForExist("#question-page-editor [id='label']");
+    browser.setValue(DesignQuestionnairePage.setAnswerTitle(), 'Test Answer 2');
+
+    // Preview and verifies the newly created questionnaire
+    browser.click(DesignQuestionnairePage.clickAddAnswer());
+    browser.click(DesignQuestionnairePage.clickPreview());
+    let tabIds = browser.getTabIds();
+    browser.switchTab(tabIds[1]);
+    expect(browser.getTitle()).toEqual('Test Question 1 - title');
```

Comment:
For the asserts we should look at using chai-as-promised to make this consistent with runner. This will wait x amount of time for the evaluation to be true. which should also make the tests more resilient.